packages/-serializer-encapsulation-test-app/tests/integration/create-record-test.js

```diff
@@ -27,10 +27,11 @@ class Person extends Model {
   }
 }
 
-module('inteegration/create-record - running createRecord with minimum serializer', function(hooks) {
+module('integration/create-record - running createRecord with minimum serializer', function(hooks) {
   setupTest(hooks);
 
   hooks.beforeEach(function(assert) {
+    this.owner.unregister('service:store');
```

Comment:
Why do we need to unregister? if we are explicitly registering (just below) and only rely on resolution (e.g. there is no earlier `owner.register('service:store')` ) this change should not be needed.

Reply:
wasn't required afterall, removed